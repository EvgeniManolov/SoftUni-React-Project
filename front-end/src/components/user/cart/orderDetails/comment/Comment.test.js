import React from 'react';
import Comment from './Comment';
import {shallow} from 'enzyme';

it('renders correctly', () => {
    const wrapper = shallow(<Comment/>)
    expect(wrapper).toMatchSnapshot();
  });


  it('check for html changes', () => {
    const wrapper = shallow(<Comment/>)
    expect(wrapper).toBeTruthy();
});