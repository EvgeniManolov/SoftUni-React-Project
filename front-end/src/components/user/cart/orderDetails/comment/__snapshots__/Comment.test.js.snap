// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Comment />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "bsClass": "row",
      "children": <Col
        bsClass="col"
        componentClass="div"
        sm={8}
>
        <FormGroup
                bsClass="form-group"
                controlId={undefined}
        >
                <ControlLabel
                        bsClass="control-label"
                        srOnly={false}
                >
                        Коментар
                </ControlLabel>
                <FormControl
                        bsClass="form-control"
                        componentClass="textarea"
                        name="comment"
                        onChange={[Function]}
                        value={undefined}
                />
        </FormGroup>
</Col>,
      "componentClass": "div",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "bsClass": "col",
        "children": <FormGroup
          bsClass="form-group"
          controlId={undefined}
>
          <ControlLabel
                    bsClass="control-label"
                    srOnly={false}
          >
                    Коментар
          </ControlLabel>
          <FormControl
                    bsClass="form-control"
                    componentClass="textarea"
                    name="comment"
                    onChange={[Function]}
                    value={undefined}
          />
</FormGroup>,
        "componentClass": "div",
        "sm": 8,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "form-group",
          "children": Array [
            <ControlLabel
              bsClass="control-label"
              srOnly={false}
>
              Коментар
</ControlLabel>,
            <FormControl
              bsClass="form-control"
              componentClass="textarea"
              name="comment"
              onChange={[Function]}
              value={undefined}
/>,
          ],
          "controlId": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "control-label",
              "children": "Коментар",
              "srOnly": false,
            },
            "ref": null,
            "rendered": "Коментар",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "form-control",
              "componentClass": "textarea",
              "name": "comment",
              "onChange": [Function],
              "value": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "bsClass": "row",
        "children": <Col
          bsClass="col"
          componentClass="div"
          sm={8}
>
          <FormGroup
                    bsClass="form-group"
                    controlId={undefined}
          >
                    <ControlLabel
                              bsClass="control-label"
                              srOnly={false}
                    >
                              Коментар
                    </ControlLabel>
                    <FormControl
                              bsClass="form-control"
                              componentClass="textarea"
                              name="comment"
                              onChange={[Function]}
                              value={undefined}
                    />
          </FormGroup>
</Col>,
        "componentClass": "div",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "col",
          "children": <FormGroup
            bsClass="form-group"
            controlId={undefined}
>
            <ControlLabel
                        bsClass="control-label"
                        srOnly={false}
            >
                        Коментар
            </ControlLabel>
            <FormControl
                        bsClass="form-control"
                        componentClass="textarea"
                        name="comment"
                        onChange={[Function]}
                        value={undefined}
            />
</FormGroup>,
          "componentClass": "div",
          "sm": 8,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "bsClass": "form-group",
            "children": Array [
              <ControlLabel
                bsClass="control-label"
                srOnly={false}
>
                Коментар
</ControlLabel>,
              <FormControl
                bsClass="form-control"
                componentClass="textarea"
                name="comment"
                onChange={[Function]}
                value={undefined}
/>,
            ],
            "controlId": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "control-label",
                "children": "Коментар",
                "srOnly": false,
              },
              "ref": null,
              "rendered": "Коментар",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "form-control",
                "componentClass": "textarea",
                "name": "comment",
                "onChange": [Function],
                "value": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": false,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
  Symbol(enzyme.__childContext__): null,
}
`;
